scripts/archive.sh

```diff
@@ -44,5 +44,5 @@ fi
 echo
 
 rm -r "$workspace_dir"
-# on TPP windows this needs to be docker 
+# on TPP windows this needs to be docker
```

Comment:
I think these two lines can go now, as they we for running it pre-vm migration

Reply:
https://github.com/opensafely-core/job-runner/pull/614